### ISSUE TITLE
Work with ssh urls of repos

### ISSUE THREAD

**Issue by k-bx** [donaldpiret/github-pivotal-flow#6]
I use ssh-links to repos (not https), and github-pivotal-flow gives an error saying it can't parse that uri (`git@github.com:user/repo.git`).

**Comment by donaldpiret** [comment]
Can't seem to reproduce this. Could you paste the relevant section of your .git/config file? There might be something special about your github user id or repo name that is failing the regexp.

**Comment by k-bx** [comment]
@donaldpiret  it seems that bug only happens if URL has uppercase letters. If you use lowercase (even for uppercase-repo) -- it works.

**Comment by k-bx** [comment]
Since merge of https://github.com/donaldpiret/github-pivotal-flow/pull/11 this should be fixed.

**Comment by k-bx** [comment]
Sorry, maybe it wasn't related after all, but it doesn't also seem to be directly related to this project. Here's an example of an error:

```
➜  tldr-data git:(develop) git publish
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/uri/common.rb:176:in `split': bad URI(is not URI?): git@github.com:ThoughtLeadr/tldr-data.git (URI::InvalidURIError)
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/uri/common.rb:211:in `parse'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/uri/common.rb:747:in `parse'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/uri/common.rb:996:in `URI'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/lib/github_pivotal_flow/project.rb:20:in `initialize'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/lib/github_pivotal_flow/configuration.rb:99:in `new'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/lib/github_pivotal_flow/configuration.rb:99:in `project'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/lib/github_pivotal_flow/configuration.rb:232:in `ensure_github_api_token'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/lib/github_pivotal_flow/configuration.rb:15:in `validate'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/lib/github_pivotal_flow/command.rb:20:in `initialize'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/bin/git-publish:6:in `new'
        from /Library/Ruby/Gems/2.0.0/gems/github-pivotal-flow-1.2/bin/git-publish:6:in `<top (required)>'
        from /usr/bin/git-publish:23:in `load'
        from /usr/bin/git-publish:23:in `<main>'
```

**Comment by regnerjr** [comment]
I am having this same issue. 
```
❯ git start                                                                                                                                              ios-three develop ✗ b812052
/Users/john/.rbenv/versions/2.2.0/lib/ruby/2.2.0/uri/rfc3986_parser.rb:66:in `split': bad URI(is not URI?): git@github.com:Tripstr/ios-three.git (URI::InvalidURIError)
        from /Users/john/.rbenv/versions/2.2.0/lib/ruby/2.2.0/uri/rfc3986_parser.rb:72:in `parse'
        from /Users/john/.rbenv/versions/2.2.0/lib/ruby/2.2.0/uri/common.rb:226:in `parse'
        from /Users/john/.rbenv/versions/2.2.0/lib/ruby/2.2.0/uri/common.rb:713:in `URI'
        from /Users/john/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/github-pivotal-flow-1.2/lib/github_pivotal_flow/project.rb:20:in `initialize'
```

.gitconfig:
```

[pivotal]
  api-token = 44aa4eea04b********************bd35
  project-id = 13****90

[gitflow "branch"]
  development = develop
[gitflow "prefix"]
  feature = feature
  hotfix = hotfix
  release = release
```

Not sure how to troubleshoot this. Let me know if there is more info I can provide.

**Comment by donaldpiret** [comment]
Hey John, I'll have some time this weekend and will take a look at this. Thank for reporting!

**Comment by regnerjr** [comment]
Let me know how I can help. I am not a ruby dev, but do know my way around. I think it might even be a dependency problem.(using ruby 2.2.0)

**Comment by donaldpiret** [comment]
Looks like I had forgotten to cut a new gem version that contained the fix for this. Should be fixed in 1.3
